.github/workflows/doc-gatekeeper.yml

```diff
@@ -0,0 +1,25 @@
+# This is a ci workflow to help you get started with Actions
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  pull_request:
+    branches: [ master ]
```

Comment:
add paths, https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths

Reply:
done